src/schoology-extractor/tests/test_schoology_extract_facade.py

```diff
@@ -242,8 +242,9 @@ def system() -> Tuple[pd.DataFrame, Mock]:
 
             # Act
             result = service.get_assignments(section_id)
+            mapped_result = service.map_assignments_to_udm(result, section_id)

```

Comment:
Can you separate out the unit tests, so that the test is only covering one function instead of both `get_assignments` and `map_assignments_to_udm`?

Reply:
I will.

Reply:
✅